pkgs/desktops/gnome-3/core/tracker-miners/default.nix

```diff
@@ -1,47 +1,82 @@
-{ stdenv, intltool, fetchurl, libxml2, upower
-, substituteAll, pkgconfig, gtk3, glib, gexiv2
-, bash, wrapGAppsHook, itstool, vala, sqlite, libxslt
-, gnome3, librsvg, gdk_pixbuf, libnotify
-, evolution-data-server, gst_all_1, poppler
-, icu, taglib, libjpeg, libtiff, giflib, libcue
-, libvorbis, flac, exempi, networkmanager
-, libpng, libexif, libgsf, libuuid, bzip2
-, libsoup, json-glib, libseccomp
-, libiptcdata }:
-
-let
+{ stdenv, fetchurl, meson, ninja, intltool, libxml2, upower, substituteAll
+, pkgconfig, gtk3, glib, bash, wrapGAppsHook, itstool, vala, sqlite
+, libxslt, gnome3, librsvg, gdk_pixbuf, libnotify , evolution-data-server
+, gst_all_1, poppler, icu, taglib, libjpeg, libtiff, giflib, libcue
+, libvorbis, flac, exempi, networkmanager , libpng, libexif, libgsf
+, libuuid, bzip2, libsoup, json-glib, libseccomp, libiptcdata, dbus
+, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "${pname}-${version}";
   pname = "tracker-miners";
   version = "2.0.4";
-in stdenv.mkDerivation rec {
-  name = "${pname}-${version}";
 
   src = fetchurl {
     url = "mirror://gnome/sources/${pname}/${gnome3.versionBranch version}/${name}.tar.xz";
     sha256 = "0mp9m2waii583sjgr61m1ni6py6dry11r0rzidgvw1g4cxhn89j6";
   };
 
-  NIX_CFLAGS_COMPILE = "-I${poppler.dev}/include/poppler";
+  LIBRARY_PATH = stdenv.lib.makeLibraryPath [ giflib ];
```

Comment:
I do not like this, how did this work before? Do we need to open an upstream bug?

Reply:
It seems that is an issue with our Meson hooks. We should check how `AC_CHECK_LIB` handles this.